src/Microsoft.Health.Dicom.Functions/Configs/DicomFunctionsConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Dicom.Core.Configs;
+
+namespace Microsoft.Health.Dicom.Operations.Functions.Configs
+{
+    /// <summary>
+    /// Represents configuration settings related to the Dicom functions.
```

Comment:
nit: `Dicom` is redundant in the name and we chose not to prefix most of our classes with it.

Reply:
It simulates [`DicomServerConfiguration`](https://github.com/microsoft/dicom-server/blob/main/src/Microsoft.Health.Dicom.Api/Configs/DicomServerConfiguration.cs)...